.github/workflows/variant_remapping.yml

```diff
@@ -0,0 +1,40 @@
+name: Variant Remapping
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        # Install Nextflow
+        mkdir /tmp/nextflow && cd /tmp/nextflow
+        wget -qO- get.nextflow.io | bash
+        echo "/tmp/nextflow" >> $GITHUB_PATH
+        cd -
```

Comment:
Nextflow is actually [available through Bioconda](https://bioconda.github.io/recipes/nextflow/README.html), so seems like this bit could be replaced by just adding it as a dependency in `conda.yml`. An additional rationale for that is that currently this step is very noisy in the logs and adds hundreds of lines like this:
![image](https://user-images.githubusercontent.com/10669118/107019824-0b787400-679a-11eb-9a13-feb6935d534a.png)

Reply:
Made the the conda install quiet with `-q`